src/components/RichTextEditor/renderer/renderBlockStyles.tsx

```diff
@@ -27,15 +35,31 @@ export const textAlignClassMap: Record<TextAlignTypes, string> = {
     [TextAlignTypes.AlignRight]: "tw-text-right",
 };
 
-export const renderBlockStyles = (props: RenderElementProps): JSX.Element => {
+export const headingTypeClassname = {
```

Comment:
I'd maybe rename this to `textStyleClassname` because it includes not only headings but also the custom and paragraph styles